src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -2852,17 +2852,19 @@ void AutomationInstrumentClipView::setParameterAutomationValue(ModelStackWithAut
 	interpolationAfter = getNodeInterpolation(modelStack, squareStart, false);
 
```

Comment:
For the node to the left, I just cleaned up the code a bit by creating a variable for the left adjacent node being edited.